app/react-native/package.json

```diff
@@ -62,6 +62,7 @@
     "emotion-theming": "^10.0.19",
     "glob": "^7.1.7",
     "prettier": "^2.4.1",
+    "react-native-safe-area-context": "^4.2.4",
```

Comment:
this must be added as a peer dependency since the native module must be auto linked by react native

Reply:
should be added to the setup guides also